src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs

```diff
@@ -0,0 +1,421 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.AnalyzerPowerPack;
+using Microsoft.AnalyzerPowerPack.Design;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Microsoft.CodeAnalysis.UnitTests;
+using Roslyn.Diagnostics.Test.Utilities;
+
+namespace Microsoft.AnalyzerPowerPack.UnitTests
+{
+    public class CA1008Tests : DiagnosticAnalyzerTestBase
+    {
+        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
+        {
+            return new CA1008DiagnosticAnalyzer();
+        }
+
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+        {
+            return new CA1008DiagnosticAnalyzer();
+        }
+
+        [WorkItem(836193, "DevDiv")]
```

Comment:
@tmeschter I see. I will update my PR to retain those WorkItems after this one merged.